### PR TITLE
Add hoverlabel.namelength widget

### PR DESCRIPTION
**Pull request by VeraZab**
closes: https://github.com/plotly/react-chart-editor/issues/890
Screen Shot 2019-04-11 at 3 26 52 PM
Screen Shot 2019-04-11 at 3 26 57 PM
Screen Shot 2019-04-11 at 3 27 04 PM
Screen Shot 2019-04-11 at 3 27 32 PM

### CODE REVIEW

**Comment by VeraZab** [comment]
ready for review ✨ review @nicolaskruchten, although I posted about a possible bug in plotly_js channel

**Comment by VeraZab** [comment]
Screen Shot 2019-04-11 at 4 09 19 PM

Actually, I'm leaving this 'Hide' option in for now for hoverlabel.namelength, because under certain scenarios: like switching from a template mode, where the namelength was hidden, and going into 'Values' mode can leave user in a weird state, where the figure has the 'hidden' option selected but it does not show in the UI. 

And there's other usercases where switching around options can leave the user in a weird state. So I'm leaving all options for now.

File: src/default_panels/StyleLayoutPanel.js [hunk 1]
```diff
@@ -184,12 +184,12 @@ const StyleLayoutPanel = (props, {localize: _}) => (
           {_(
             'You can refer to the items in this column in any text fields of the editor like so: '
           )}
-          <p>
-            {_('Ex: ')}
-            <span style={{letterSpacing: '1px', fontStyle: 'italic', userSelect: 'text'}}>
-              {_('My custom title %{meta[1]}')}
-            </span>
-          </p>
```
Comment (dmt0) [inline src/default_panels/StyleLayoutPanel.js]: 👍

File: src/default_panels/StyleLayoutPanel.js [hunk 2]
```diff
@@ -184,12 +184,12 @@ const StyleLayoutPanel = (props, {localize: _}) => (
...
+        </p>
+        <p>
+          {_('Ex: ')}
+          <span style={{letterSpacing: '1px', fontStyle: 'italic', userSelect: 'text'}}>
+            {_('My custom title %{meta[1]}')}
+          </span>
```
Comment (dmt0) [inline src/default_panels/StyleLayoutPanel.js]: 👍

**Comment by dmt0** [comment]
Yay! 💃

**Pull request merged**